### DISCUSSION TITLE
UnstableRate-bar always visible regardless if overlay is hidden ?

### DISCUSSION THREAD

**Discussion by nyaruku** [ppy/osu#31121] [Ideas]
I'm missing the stable feature in lazer, where the UnstableRate-bar is always visible, even if you hide the overlay elements, it's possible to achieve this through the "skin layout editor", but you need to do this for every skin manually, because you also can't duplicate skin layouts to other skins.

**Comment by fleuph** [comment]
Sorry you have been waiting so long for a reply.

Yes, you have to do it manually for every skin, but it is possible to speed the process up by copying and pasting (using standard keyboard shortcuts) from one skin to another. One potential option is to, in the default imported classic skin placement, place the hit error meter in the playfield layer on purpose. But unless you have dozens upon dozens of skins the current solution is likely fine for those who want to hide their HUD.
